client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js

```diff
@@ -0,0 +1,263 @@
+			),
+		},
+	},
+	store: {
+		all: {
+			title: translate( 'Store WordPress Themes' ),
+			header: translate( 'Find the perfect theme for your online store.' ),
+			description: translate(
+				"Start your online store with the perfect theme. Choose from options for WordPress and WooCommerce sites. Then, activate the one that's right for you."
+			),
+		},
+		free: {
+			title: translate( 'Free Store WordPress Themes' ),
+			header: translate( 'Discover the perfect free theme for your online store.' ),
+			description: translate(
+				"Launch your online store with the perfect free theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		premium: {
+			title: translate( 'Premium Store WordPress Themes' ),
+			header: translate( 'Choose premium themes for your online store.' ),
+			description: translate(
+				"Enhance your online store with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
+			),
+		},
+		marketplace: {
+			title: translate( 'Paid Store WordPress Themes' ),
+			header: translate( 'Select paid themes for your online store.' ),
+			description: translate(
+				"Improve your online store with the perfect paid theme. Browse high-quality, professionally-designed options. Then, purchase the one that's right for you."
+			),
+		},
+	},
+	about: {
+		all: {
+			title: translate( 'About WordPress Themes' ),
+			header: translate( 'Find the perfect theme for your about website.' ),
+			description: translate(
+				"Start your about website with the perfect theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		free: {
+			title: translate( 'Free About WordPress Themes' ),
+			header: translate( 'Discover the perfect free theme for your about website.' ),
+			description: translate(
+				"Launch your about website with the perfect free theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		premium: {
+			title: translate( 'Premium About WordPress Themes' ),
+			header: translate( 'Choose premium themes for your about website.' ),
+			description: translate(
+				"Enhance your about website with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
+			),
+		},
+		marketplace: {
+			title: translate( 'Paid About WordPress Themes' ),
+			header: translate( 'Select paid themes for your about website.' ),
+			description: translate(
+				"Improve your about website with the perfect paid theme. Browse high-quality, professionally-designed options. Then, purchase the one that's right for you."
+			),
+		},
+	},
+	'link-in-bio': {
+		all: {
+			title: translate( 'Link in Bio WordPress Themes' ),
+			header: translate( 'Find the perfect theme for your about website.' ),
```

Comment:
This doesn't match the spreadsheet
```suggestion
			header: translate( 'Find the perfect theme for your link in bio website.' ),
```